### DISCUSSION TITLE
[React] Two or more Reusable Floating Component

### DISCUSSION THREAD

**Discussion by humanscape-jin** [floating-ui/floating-ui#2259] [React DOM] [answered]
Is there a way to show more than two Reusable Floating Components?
For example, when I click the My Trigger button in the Reusable Popover Component Sandbox in React Example,
I want to show Reusable Tooltip under the Reusable Popover (or opposite case)

**Comment by atomiks** [comment, marked as answer]
So you want to make the popover itself the reference of the tooltip? Making the `PopoverContent` the `TooltipTrigger` should work (with `asChild` prop)

**Reply by humanscape-jin** [reply]
thanks a lot :)
